extensions/interactions/base.py

```diff
@@ -143,6 +143,25 @@ def _stats_log_template(self):
             return '{{answer}}'
 
     @property
+    def rules(self):
```

Comment:
That's reasonable, although I think it actually supports the suggestion I made (but for a different reason), since you're returning a dict here and not a list. If 'rules' suggests a list then the method name is problematic.